app/templates/events/event_list.html

```diff
@@ -57,34 +57,34 @@ <h1 class="text-center">Events List for {{selectedTerm.description}}</h1>
   <table class="table table-striped">
     <thead>
       <tr>
+        {% if type != "bonner" %}
+        <th>Program</th>
+        {% endif %}
         <th scope="col">Event Name</th>
         <th scope="col">Date</th>
         <th scope="col">Time</th>
```

Comment:
Maybe consider changing this to Start Time instead of just Time?

Reply:
Since that column has both start time and end time I think it would be fine to keep it as it is.

Reply:
that's true. I meant to request this change for the userProfile page since we only have the start time there.